lib/utility/include/WatchdogWrapper.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+namespace Utility {
+
+class WatchdogWrapper;
+
+class WatchdogWrapper {
+ public:
+  static void startWatchdog(uint32_t countdown_ms = 5000, std::chrono::milliseconds pet_ms = 1000ms);
```

Comment:
It would be nice if we can pass in an std chrono duration type for countdown_ms instead of uint32_t